Documentation/ApiOverview/BackendModules/BackendModuleApi/Index.rst

```diff
@@ -40,7 +40,7 @@ using the following API:
         ],
         [
             'access' => 'admin',
-            'icon' => 'EXT:beuser/Resources/Public/Icons/module-beuser.svg',
+            'iconIdentifier' => 'module-beuser',
```

Comment:
Since `icon` still works you could add a note that it should only be used for custom icons in own packages. For core icons the `iconIdentifier` should be used as you have noticed.

Reply:
IMHO also for extensions it's cleaner to register the icons at the IconRegistry and use the identifiers. But agree the information about `icon` should not get lost as it's still valid.